src/oaklib/cli.py

```diff
@@ -2051,7 +2051,7 @@ def add_taxon_constraints(constraints, evolution_file, predicates: List, output)
         impl.enable_transitive_query_cache()
         for st in sts:
             try:
-                st = test_candidate_taxon_constraint(impl, st, predicates=actual_predicates)
+                st = eval_candidate_taxon_constraint(impl, st, predicates=actual_predicates)
```

Comment:
this is leaking over from a different PR?

Reply:
No I manually made this change when I discovered it working with Justin and Harry.